<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseCard</class>
 <widget class="QWidget" name="CourseCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>675</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="course_box">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>500</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QPushButton" name="enter_class">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>140</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    border-radius: 15px;
    padding-left: 10px;
    padding-right: 10px;
    color: rgb(255,255,255);
    min-width: 120px;
    background: qlineargradient(x1 : 0, x2 : 1, y1: 0, y2: 1, stop : 0 rgb(18,111,237), stop : 1 rgb(45,147,250));
}

QPushButton:hover {
    background: qlineargradient(x1 : 0, x2 : 1, y1: 0, y2: 1, stop : 0 rgb(3,92,227), stop:1 rgb(21,133,244));
}

QPushButton:pressed {
    background: qlineargradient(x1 : 0, x2 : 1, y1: 0, y2: 1, stop : 0 rgb(0,77,201), stop : 1 rgb(4,117,231));
}

QPushButton:disabled {
    background: qlineargradient(x1 : 0, x2 : 1, y1: 0, y2: 1, stop : 0 rgb(196,196,196), stop : 1 rgb(207,207,207));
}</string>
    </property>
    <property name="text">
     <string>进入课堂</string>
    </property>
   </widget>
   <widget class="QPushButton" name="course_info">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>140</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    border-radius: 15px;
    padding-left: 10px;
    padding-right: 10px;
    color: rgb(255,255,255);
    min-width: 120px;
    background: qlineargradient(x1 : 0, x2 : 1, y1: 0, y2: 1, stop : 0 rgb(18,111,237), stop : 1 rgb(45,147,250));
}

QPushButton:hover {
    background: qlineargradient(x1 : 0, x2 : 1, y1: 0, y2: 1, stop : 0 rgb(3,92,227), stop:1 rgb(21,133,244));
}

QPushButton:pressed {
    background: qlineargradient(x1 : 0, x2 : 1, y1: 0, y2: 1, stop : 0 rgb(0,77,201), stop : 1 rgb(4,117,231));
}

QPushButton:disabled {
    background: qlineargradient(x1 : 0, x2 : 1, y1: 0, y2: 1, stop : 0 rgb(196,196,196), stop : 1 rgb(207,207,207));
}</string>
    </property>
    <property name="text">
     <string>课程信息</string>
    </property>
   </widget>
   <widget class="QFrame" name="course_card_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>300</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#d0a8a9</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="course_status">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>课程状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="course_name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华光超粗黑_CNKI</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>课程名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {	
	background-color: #ca9d9e;
	border: 2px solid rgb(60, 60, 60);
	border-radius: 5px;
}
QPushButton:hover {	
	background-color: #e7d3d4;
	border: 2px solid rgb(70, 70, 70);
}
QPushButton:pressed {	
	background-color: #dcbebe;
	border: 2px solid rgb(255, 165, 24);	
	color: rgb(35, 35, 0);
}</string>
     </property>
     <property name="text">
      <string>结课</string>
     </property>
    </widget>
   </widget>
   <zorder>course_card_background</zorder>
   <zorder>enter_class</zorder>
   <zorder>course_info</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
